web/src/components/tasks/anytime-action/tax-clearance-certificate/AnytimeActionTaxClearanceCertificate.tsx

```diff
@@ -0,0 +1,205 @@
+import { Heading } from "@/components/njwds-extended/Heading";
+import { getInitialData } from "@/components/tasks/anytime-action/tax-clearance-certificate/helpers";
+import { TaxClearanceSteps } from "@/components/tasks/anytime-action/tax-clearance-certificate/TaxClearanceSteps";
+import { AddressContext } from "@/contexts/addressContext";
+import {
+  createDataFormErrorMap,
+  DataFormErrorMapContext,
+} from "@/contexts/dataFormErrorMapContext";
+import { ProfileDataContext } from "@/contexts/profileDataContext";
+import { TaxClearanceCertificateDataContext } from "@/contexts/taxClearanceCertificateDataContext";
+import { useFormContextHelper } from "@/lib/data-hooks/useFormContextHelper";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { AnytimeActionLicenseReinstatement, AnytimeActionTask } from "@/lib/types/types";
+import { getFlow, scrollToTop, useMountEffectWhenDefined } from "@/lib/utils/helpers";
+import { emptyTaxClearanceCertificateData } from "@businessnjgovnavigator/shared";
+import {
+  emptyFormationAddressData,
+  FormationAddress,
+} from "@businessnjgovnavigator/shared/formationData";
+import { emptyProfileData, ProfileData } from "@businessnjgovnavigator/shared/profileData";
+import { Dispatch, ReactElement, SetStateAction, useState } from "react";
+
+const pickData = <T extends object, K extends keyof T>(obj: T, keys: K[]): Pick<T, K> => {
```

Comment:
dust - I think this file should liev in `@/contexts/taxClearanceCertificateDataContext`, it would be easier to find from that file.

Reply:
Sure. We can move this.